scripts/upload-image.sh

```diff
@@ -15,14 +13,13 @@ echo DESC: $DESC
 
 # enviroment variables
 SEABUG_ADMIN_TOKEN=ADMIN_TOKEN
-export PGPASSWORD=seabug
 
 ############################################################
 # Prepare
 ############################################################
 
-# Setup server admin token, password: seabug
-psql -U seabug -h localhost -q -c "INSERT INTO admin_token(token) VALUES ('$SEABUG_ADMIN_TOKEN') ON CONFLICT DO NOTHING"
+# Setup server admin token
+sudo -u nft-marketplace-server psql -q -c "INSERT INTO admin_token(token) VALUES ('$SEABUG_ADMIN_TOKEN') ON CONFLICT DO NOTHING"
```

Comment:
i don't like this. maybe the whole script should be run with that user. or create a systemd oneshot service.

Reply:
I don't know, consider that this script is present only in the vm, not in the deployment nixos configuration. I think is only a convenient utility used during development since users cannot add new NFT in UI for now. When in the future this will be implemented I think that this script will become useless.

@rynoV am I right? What do you think?

Reply:
> I think is only a convenient utility used during development since users cannot add new NFT in UI for now. When in the future this will be implemented I think that this script will become useless.

This is accurate